projects/ui/src/lib/components/po-navbar/po-navbar.component.spec.ts

```diff
@@ -387,6 +387,15 @@ describe('PoNavbarComponent:', () => {
       expect(component['changeNavbarMenuItems']).not.toHaveBeenCalled();
     });
 
+    it(`initNavbarMenu: should call 'initNavbarMenu'`, () => {
```

Comment:
Aqui não seria: "should call 'validateMenuLogo'"?

Reply:
Isso mesmo.